test/app.test.js

```diff
@@ -45,6 +45,22 @@ describe('Main Generator', function () {
     it('should generate a package.json file', function () {
       assert.file('temp/package.json');
     });
+
+    it('should have populated package.json correctly', function (done) {
+      fs.readFile('temp/package.json', function (err, data) {
```

Comment:
We can just assert the file content, which should be better than reading the file.